### PR TITLE
DT-493 SVG stop layer

### PR DESCRIPTION
**Pull request by hannesj**

### CODE REVIEW

**Comment by hannesj** [comment]
Requires HSLdevcom/digitransit-deploy#73

File: app/component/map/stop/stop-marker-tile-layer.cjsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+provideContext = require 'fluxible-addons-react/provideContext'
+intl          = require 'react-intl'
+
+class SVGTile
+  constructor: (@coords, done, @map) ->
+    @el = document.createElementNS 'http://www.w3.org/2000/svg', 'svg'
+    @el.setAttribute "class", "leaflet-tile"
+    @el.setAttribute "viewBox", "0 0 4096 4096"
+    @el.addEventListener "click", @onMapClick
+    if @coords.z < 14
```
Comment (samuliheljo) [inline app/component/map/stop/stop-marker-tile-layer.cjsx]: Could we check this before creating  SVGTile? I mean, is there a reason to create empty SVGTile if zoom level is low?

Comment (hannesj) [inline app/component/map/stop/stop-marker-tile-layer.cjsx]: Leaflet doesn't like if we don't return any element to be shown. Thus generate an empty SCG to show.

**Comment by hannesj** [comment]
All issues found should be fixed now

**Pull request merged**